src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -735,6 +735,156 @@ RppStatus rppt_ricap_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** pixelate ********************/
+
+RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
+                             RpptDescPtr srcDescPtr,
+                             RppPtr_t dstPtr,
+                             RpptDescPtr dstDescPtr,
+                             Rpp32f pixelationPercentage,
+                             RpptROIPtr roiTensorPtrSrc,
+                             RpptRoiType roiType,
+                             rppHandle_t rppHandle)
+{
+    RppLayoutParams srcLayoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+    RpptDescPtr interDescPtr = srcDescPtr;
+    RpptImagePatchPtr internalDstImgSizes = static_cast<RpptImagePatch *> (calloc(dstDescPtr->n, sizeof(RpptImagePatch)));
+    RpptROI *internalRoiTensorPtrSrc = static_cast<RpptROI *>( calloc(dstDescPtr->n, sizeof(RpptROI)));
+
+    for(int i = 0 ; i < dstDescPtr->n ; i++)
+    {
+        internalDstImgSizes[i].width = internalRoiTensorPtrSrc[i].xywhROI.roiWidth = (roiTensorPtrSrc[i].xywhROI.roiWidth * (100 - pixelationPercentage)) / 100;
+        internalDstImgSizes[i].height = internalRoiTensorPtrSrc[i].xywhROI.roiHeight = (roiTensorPtrSrc[i].xywhROI.roiHeight * (100 - pixelationPercentage)) / 100;
+    }
+
+    Rpp64u interBufferSize = static_cast<Rpp64u>(interDescPtr->strides.nStride) * static_cast<Rpp64u>(interDescPtr->n);
+
+    if ((srcDescPtr->dataType == RpptDataType::U8) && (dstDescPtr->dataType == RpptDataType::U8))
+    {
+        Rpp8u *interDstPtr = static_cast<Rpp8u *>(calloc(interBufferSize, sizeof(Rpp8u)));
+        resize_bilinear_u8_u8_host_tensor(static_cast<Rpp8u*>(srcPtr) + srcDescPtr->offsetInBytes,
+                                          srcDescPtr,
+                                          static_cast<Rpp8u*>(interDstPtr),
+                                          interDescPtr,
```

Comment:
Is interDescPtr changing anywhere in the whole pipeline? If not, pls just use srcDescPtr

Reply:
Done

Reply:
I thought interDescPtr was different from srcDescPtr as per last call

Reply:
for Host this interDescPtr is not used